# Foundations of Computer Science 2023 Project

We first import useful packages

In [1]:
import pandas as pd
import numpy as np
import re

## 1. Extract all trips with busRoute 83

First of all we need to read the dataset

In [2]:
trips=pd.read_csv('metaData.csv')

Let's check what the dataset is like

In [3]:
trips

name  busNumber  \
0     B183_2019-04-30_03-18-56_2019-04-30_08-44-20        183   
1     B183_2019-04-30_13-22-07_2019-04-30_17-54-02        183   
2     B183_2019-05-01_05-58-51_2019-05-01_22-32-30        183   
3     B183_2019-05-03_02-50-21_2019-05-03_05-53-20        183   
4     B183_2019-05-03_15-41-57_2019-05-03_23-06-24        183   
...                                            ...        ...   
1404  B208_2022-12-06_14-43-49_2022-12-06_18-22-52        208   
1405  B208_2022-12-07_05-13-02_2022-12-07_19-19-53        208   
1406  B208_2022-12-08_05-22-20_2022-12-08_18-39-15        208   
1407  B208_2022-12-09_23-55-12_2022-12-10_03-24-28        208   
1408  B208_2022-12-10_23-34-46_2022-12-11_03-29-05        208   

             startTime_iso  startTime_unix           endTime_iso  \
0     2019-04-30T03:18:56Z      1556594336  2019-04-30T08:44:20Z   
1     2019-04-30T13:22:07Z      1556630527  2019-04-30T17:54:02Z   
2     2019-05-01T05:58:51Z      1556690331  2019-05-01T22:32:30Z   
3     2019-05-03T02:50:21Z      1556851821  2019-05-03T05:53:20Z   
4     2019-05-03T15:41:57Z      1556898117  2019-05-03T23:06:24Z   
...                    ...             ...                   ...   
1404  2022-12-06T14:43:49Z      1670337829  2022-12-06T18:22:52Z   
1405  2022-12-07T05:13:02Z      1670389982  2022-12-07T19:19:53Z   
1406  2022-12-08T05:22:20Z      1670476940  2022-12-08T18:39:15Z   
1407  2022-12-09T23:55:12Z      1670630112  2022-12-10T03:24:28Z   
1408  2022-12-10T23:34:46Z      1670715286  2022-12-11T03:29:05Z   

      endTime_unix  drivenDistance busRoute  energyConsumption  \
0       1556613860        77213.87        -       4.785852e+08   
1       1556646842        59029.60       31       4.022585e+08   
2       1556749950       240900.40       33       1.445733e+09   
3       1556862800        42565.48        -       2.819867e+08   
4       1556924784       125277.20       72       6.207258e+08   
...            ...             ...      ...                ...   
1404    1670350972        51798.78       32       4.260419e+08   
1405    1670440793       210041.60       83       1.536697e+09   
1406    1670524755       190372.70       83       1.415700e+09   
1407    1670642668        59548.57       N1       4.519165e+08   
1408    1670729345        72293.83       N2       5.350956e+08   

      itcs_numberOfPassengers_mean  itcs_numberOfPassengers_min  \
0                         5.538860                            0   
1                        33.114580                            4   
2                        19.689140                            0   
3                         1.685185                            0   
4                        23.753570                            1   
...                            ...                          ...   
1404                     39.808990                            0   
1405                     28.785390                            0   
1406                     29.879400                            0   
1407                     20.105260                            0   
1408                     15.993200                            0   

      itcs_numberOfPassengers_max  status_gridIsAvailable_mean  \
0                              20                     0.740640   
1                              74                     0.855234   
2                              55                     0.777860   
3                               8                     0.767122   
4                              67                     0.907342   
...                           ...                          ...   
1404                           83                     0.739349   
1405                          115                     0.434858   
1406                          102                     0.439916   
1407                           74                     0.495739   
1408                           52                     0.459492   

      temperature_ambient_mean  temperature_ambient_min  

Let us check what the different routes were

In [4]:
trips['busRoute']

0        -
1       31
2       33
3        -
4       72
        ..
1404    32
1405    83
1406    83
1407    N1
1408    N2
Name: busRoute, Length: 1409, dtype: object

Let's see if it recognizes 83 

In [5]:
trips['busRoute']==83

0       False
1       False
2       False
3       False
4       False
        ...  
1404    False
1405    False
1406    False
1407    False
1408    False
Name: busRoute, Length: 1409, dtype: bool

Since it doesn't (see lines 1405-1406) and there are routes named with letters, the data might be stored in strings

In [6]:
trips['busRoute']=='83'

0       False
1       False
2       False
3       False
4       False
        ...  
1404    False
1405     True
1406     True
1407    False
1408    False
Name: busRoute, Length: 1409, dtype: bool

Ok, that seems to be the case.

In [7]:
trips[trips['busRoute']=='83']

name  busNumber  \
154   B183_2020-03-03_04-42-38_2020-03-03_19-44-51        183   
155   B183_2020-03-06_04-53-23_2020-03-06_19-44-42        183   
157   B183_2020-03-09_14-16-13_2020-03-09_19-34-17        183   
158   B183_2020-03-10_04-50-03_2020-03-10_19-51-25        183   
159   B183_2020-03-12_04-56-41_2020-03-12_19-44-57        183   
...                                            ...        ...   
1399  B208_2022-11-30_04-47-53_2022-11-30_19-50-22        208   
1400  B208_2022-12-01_05-19-41_2022-12-01_18-20-57        208   
1401  B208_2022-12-02_04-47-48_2022-12-02_19-40-01        208   
1405  B208_2022-12-07_05-13-02_2022-12-07_19-19-53        208   
1406  B208_2022-12-08_05-22-20_2022-12-08_18-39-15        208   

             startTime_iso  startTime_unix           endTime_iso  \
154   2020-03-03T04:42:38Z      1583210558  2020-03-03T19:44:51Z   
155   2020-03-06T04:53:23Z      1583470403  2020-03-06T19:44:42Z   
157   2020-03-09T14:16:13Z      1583763373  2020-03-09T19:34:17Z   
158   2020-03-10T04:50:03Z      1583815803  2020-03-10T19:51:25Z   
159   2020-03-12T04:56:41Z      1583989001  2020-03-12T19:44:57Z   
...                    ...             ...                   ...   
1399  2022-11-30T04:47:53Z      1669783673  2022-11-30T19:50:22Z   
1400  2022-12-01T05:19:41Z      1669871981  2022-12-01T18:20:57Z   
1401  2022-12-02T04:47:48Z      1669956468  2022-12-02T19:40:01Z   
1405  2022-12-07T05:13:02Z      1670389982  2022-12-07T19:19:53Z   
1406  2022-12-08T05:22:20Z      1670476940  2022-12-08T18:39:15Z   

      endTime_unix  drivenDistance busRoute  energyConsumption  \
154     1583264691       225047.90       83       1.544278e+09   
155     1583523882       224512.30       83       1.631816e+09   
157     1583782457        77824.36       83       5.406013e+08   
158     1583869885       225095.80       83       1.692171e+09   
159     1584042297       224181.20       83       1.145860e+09   
...            ...             ...      ...                ...   
1399    1669837822       223165.00       83       1.560888e+09   
1400    1669918857       190196.00       83       1.418847e+09   
1401    1670010001       224473.40       83       1.611150e+09   
1405    1670440793       210041.60       83       1.536697e+09   
1406    1670524755       190372.70       83       1.415700e+09   

      itcs_numberOfPassengers_mean  itcs_numberOfPassengers_min  \
154                       23.47531                            0   
155                       17.41578                            0   
157                       23.18182                            0   
158                       20.96410                            0   
159                       17.21235                            0   
...                            ...                          ...   
1399                      27.89066                            2   
1400                      26.03927                            0   
1401                      24.80384                            2   
1405                      28.78539                            0   
1406                      29.87940                            0   

      itcs_numberOfPassengers_max  status_gridIsAvailable_mean  \
154                           118                     0.472180   
155                            69                     0.451028   
157                            74                     0.460099   
158                            86                     0.475233   
159                            80                     0.340882   
...                           ...                          ...   
1399                          100                     0.456196   
1400                           96                     0.450413   
1401                           91                     0.438693   
1405                          115                     0.434858   
1406                          102                     0.439916   

      temperature_ambient_mean  temperature_ambient_min  

## 2. Extract all trips where busRoute is not a number

Let us define a function to check if the route is not a number. We have to do this because some routes are just missing (see lines 0 or 3 as example) and some others are letters and numbers (such as 1407-1408)

In [8]:
def check_not_number(route):
    m=re.match('^\d+\d$' ,route)
    if m:
        return False
    else: return True

Let's test it on some strings 

In [9]:
l='lol'
a='83'
b='N1'
c='189492749274398273'
d='18273t87217hh198718198'

In [10]:
check_not_number(l)

True

In [11]:
check_not_number(a)

False

In [12]:
#check_not_number(p)

In [13]:
check_not_number(b)

True

In [14]:
check_not_number(c)

False

In [15]:
check_not_number(d)

True

In [16]:
last_one='12N3'

In [17]:
check_not_number(last_one)

True

Since it seems to work we can use it on the actual dataset

In [18]:
trips[trips['busRoute'].apply(check_not_number)==True]

name  busNumber  \
0     B183_2019-04-30_03-18-56_2019-04-30_08-44-20        183   
3     B183_2019-05-03_02-50-21_2019-05-03_05-53-20        183   
9     B183_2019-05-10_03-16-11_2019-05-10_18-51-37        183   
10    B183_2019-05-13_03-10-23_2019-05-13_23-16-13        183   
19    B183_2019-05-24_02-52-47_2019-05-24_22-35-11        183   
...                                            ...        ...   
1373  B208_2022-10-21_22-38-32_2022-10-22_02-42-21        208   
1374  B208_2022-10-22_22-34-45_2022-10-23_02-29-59        208   
1394  B208_2022-11-25_23-35-16_2022-11-26_03-30-39        208   
1407  B208_2022-12-09_23-55-12_2022-12-10_03-24-28        208   
1408  B208_2022-12-10_23-34-46_2022-12-11_03-29-05        208   

             startTime_iso  startTime_unix           endTime_iso  \
0     2019-04-30T03:18:56Z      1556594336  2019-04-30T08:44:20Z   
3     2019-05-03T02:50:21Z      1556851821  2019-05-03T05:53:20Z   
9     2019-05-10T03:16:11Z      1557458171  2019-05-10T18:51:37Z   
10    2019-05-13T03:10:23Z      1557717023  2019-05-13T23:16:13Z   
19    2019-05-24T02:52:47Z      1558666367  2019-05-24T22:35:11Z   
...                    ...             ...                   ...   
1373  2022-10-21T22:38:32Z      1666391912  2022-10-22T02:42:21Z   
1374  2022-10-22T22:34:45Z      1666478085  2022-10-23T02:29:59Z   
1394  2022-11-25T23:35:16Z      1669419316  2022-11-26T03:30:39Z   
1407  2022-12-09T23:55:12Z      1670630112  2022-12-10T03:24:28Z   
1408  2022-12-10T23:34:46Z      1670715286  2022-12-11T03:29:05Z   

      endTime_unix  drivenDistance busRoute  energyConsumption  \
0       1556613860        77213.87        -       4.785852e+08   
3       1556862800        42565.48        -       2.819867e+08   
9       1557514297       210577.00        -       1.303391e+09   
10      1557789373       267033.80        -       1.647432e+09   
19      1558737311       263432.60        -       1.448057e+09   
...            ...             ...      ...                ...   
1373    1666406541        78567.16       N1       4.347766e+08   
1374    1666492199        73427.97       N2       3.997737e+08   
1394    1669433439        72911.26       N2       4.475534e+08   
1407    1670642668        59548.57       N1       4.519165e+08   
1408    1670729345        72293.83       N2       5.350956e+08   

      itcs_numberOfPassengers_mean  itcs_numberOfPassengers_min  \
0                         5.538860                            0   
3                         1.685185                            0   
9                         8.230483                            0   
10                        7.891652                            0   
19                        7.520249                            0   
...                            ...                          ...   
1373                     16.333330                            0   
1374                     17.710530                            0   
1394                     11.216670                            1   
1407                     20.105260                            0   
1408                     15.993200                            0   

      itcs_numberOfPassengers_max  status_gridIsAvailable_mean  \
0                              20                     0.740640   
3                               8                     0.767122   
9                              43                     0.740927   
10                             45                     0.804191   
19                             44                     0.761068   
...                           ...                          ...   
1373                           45                     0.431852   
1374                           57                     0.443358   
1394                           32                     0.465024   
1407                           74                     0.495739   
1408                           52                     0.459492   

      temperature_ambient_mean  temperature_ambient_min  

## 3. For each (busNumber, busRoute) pair, determine the number of trips

Let's check how many different buses number and routes we have:

In [19]:
trips['busNumber'].unique()

array([183, 208])

In [20]:
trips['busRoute'].unique()

array(['-', '31', '33', '72', '46', '32', '83', 'N4', 'N2', 'N1'],
      dtype=object)

Now we can just group by Number and Route and check how many lines we count

In [21]:
trips.groupby(['busNumber', 'busRoute']).size()

busNumber  busRoute
183        -            11
           31           12
           32           12
           33          130
           46          104
           72          114
           83          441
           N1           10
           N2           19
           N4           11
208        31            5
           32           14
           33           25
           46           19
           72           44
           83          405
           N1            6
           N2           20
           N4            7
dtype: int64

This is in accord with point 2 of this project: the total number of trips for Route 83 is 441 for bus 183 and 405 for bus 208; that is a total of 846 trips, as the lines resulting in "trips[trips['busRoute']=='83']"

## 4. For each trip, compute the ratio between the energy consumption and the average number of passengers 

First of all let's check the data type of energyConsumption: if it is a string we'd have to convert it to a number

In [22]:
type(trips['energyConsumption'][0])

numpy.float64

No work needed here, we can just add a column with the requested information

In [23]:
trips['EnergyRatio']=trips['energyConsumption']/trips['itcs_numberOfPassengers_mean']

In [24]:
trips['EnergyRatio']

0       8.640500e+07
1       1.214747e+07
2       7.342794e+07
3       1.673328e+08
4       2.613190e+07
            ...     
1404    1.070215e+07
1405    5.338462e+07
1406    4.738047e+07
1407    2.247753e+07
1408    3.345769e+07
Name: EnergyRatio, Length: 1409, dtype: float64

## 5. For each station (itcs_stopName), determine the average number of passengers. 

itcs_stopName does not ring a bell but let's check if it is in the columns

In [25]:
trips.columns

Index(['name', 'busNumber', 'startTime_iso', 'startTime_unix', 'endTime_iso',
       'endTime_unix', 'drivenDistance', 'busRoute', 'energyConsumption',
       'itcs_numberOfPassengers_mean', 'itcs_numberOfPassengers_min',
       'itcs_numberOfPassengers_max', 'status_gridIsAvailable_mean',
       'temperature_ambient_mean', 'temperature_ambient_min',
       'temperature_ambient_max', 'EnergyRatio'],
      dtype='object')

let's check if by chance it's in the other dataset...

In [26]:
test=pd.read_csv('ZTBus_Compressed/B183_2019-04-30_03-18-56_2019-04-30_08-44-20.csv')

In [27]:
test

time_iso   time_unix  electric_powerDemand  gnss_altitude  \
0      2019-04-30T03:18:56Z  1556594336            -13.845510            NaN   
1      2019-04-30T03:18:57Z  1556594337             -3.849362            NaN   
2      2019-04-30T03:18:58Z  1556594338             -0.672331            NaN   
3      2019-04-30T03:18:59Z  1556594339             -1.087931            NaN   
4      2019-04-30T03:19:00Z  1556594340             -0.811985            NaN   
...                     ...         ...                   ...            ...   
19520  2019-04-30T08:44:16Z  1556613856             -0.254938            NaN   
19521  2019-04-30T08:44:17Z  1556613857             -0.131183            NaN   
19522  2019-04-30T08:44:18Z  1556613858             -0.131183            NaN   
19523  2019-04-30T08:44:19Z  1556613859             -0.258701            NaN   
19524  2019-04-30T08:44:20Z  1556613860             -0.191862            NaN   

       gnss_course  gnss_latitude  gnss_longitude itcs_busRoute  \
0              NaN            NaN             NaN             -   
1              NaN            NaN             NaN             -   
2              NaN            NaN             NaN             -   
3              NaN            NaN             NaN             -   
4              NaN            NaN             NaN             -   
...            ...            ...             ...           ...   
19520          NaN            NaN             NaN             -   
19521          NaN            NaN             NaN             -   
19522          NaN            NaN             NaN             -   
19523          NaN            NaN             NaN             -   
19524          NaN            NaN             NaN             -   

       itcs_numberOfPassengers itcs_stopName  ...  odometry_wheelSpeed_mr  \
0                          NaN             -  ...                     0.0   
1                          NaN             -  ...                     0.0   
2                          NaN             -  ...                     0.0   
3                          NaN             -  ...                     0.0   
4                          NaN             -  ...                     0.0   
...                        ...           ...  ...                     ...   
19520                      NaN             -  ...                     0.0   
19521                      NaN             -  ...                     0.0   
19522                      NaN             -  ...                     0.0   
19523                      NaN             -  ...                     0.0   
19524                      NaN             -  ...                     0.0   

       odometry_wheelSpeed_rl  odometry_wheelSpeed_rr  status_doorIsOpen  \
0                         0.0                     0.0                  1   
1                         0.0                     0.0                  1   
2                         0.0                     0.0                  1   
3                         0.0                     0.0                  1   
4                         0.0                     0.0                  1   
...                       ...                     ...                ...   
19520                     0.0                     0.0                  1   
19521                     0.0                     0.0                  1   
19522                     0.0                     0.0                  1   
19523                     0.0                     0.0                  1   
19524                     0.0                     0.0                  1   

       status_gridIsAvailable  status_haltBrakeIsActive  \
0                           1                         0   
1                           1                         0   
2                           1                         0   
3                           1                         0   
4                           1                         0   
...                       ...                       ...   
19

In [28]:
test['itcs_stopName'].unique()

array(['-', 'Zürich, Herdernstrasse', 'Zürich, Hardplatz',
       'Zürich, Güterbahnhof', 'Zürich, Bäckeranlage',
       'Zürich, Militär-/Langstrasse', 'Zürich, Kanonengasse',
       'Zürich, Sihlpost / HB', 'Zürich, Löwenplatz',
       'Zürich, Bahnhofplatz/HB', 'Zürich, Central', 'Zürich, Neumarkt',
       'Zürich, Kunsthaus', 'Zürich, Sprecherstrasse',
       'Zürich, Kreuzplatz', 'Zürich, Signaustrasse',
       'Zürich, Hegibachplatz', 'Zürich, Botanischer Garten',
       'Zürich, Höschgasse', 'Zürich, Fröhlichstrasse',
       'Zürich, Wildbachstrasse', 'Zürich, Bahnhof Tiefenbrunnen',
       'Zürich, Freiestrasse', 'Zürich, Klusplatz',
       'Zürich, Hölderlinsteig', 'Zürich, Klosbach', 'Zürich, Hofstrasse',
       'Zürich, Kirche Fluntern', 'Zürich, Hinterbergstrasse',
       'Zürich, Spyriplatz', 'Zürich, Bethanien', 'Zürich, Toblerplatz'],
      dtype=object)

In [29]:
nome=trips['name']
nome

0       B183_2019-04-30_03-18-56_2019-04-30_08-44-20
1       B183_2019-04-30_13-22-07_2019-04-30_17-54-02
2       B183_2019-05-01_05-58-51_2019-05-01_22-32-30
3       B183_2019-05-03_02-50-21_2019-05-03_05-53-20
4       B183_2019-05-03_15-41-57_2019-05-03_23-06-24
                            ...                     
1404    B208_2022-12-06_14-43-49_2022-12-06_18-22-52
1405    B208_2022-12-07_05-13-02_2022-12-07_19-19-53
1406    B208_2022-12-08_05-22-20_2022-12-08_18-39-15
1407    B208_2022-12-09_23-55-12_2022-12-10_03-24-28
1408    B208_2022-12-10_23-34-46_2022-12-11_03-29-05
Name: name, Length: 1409, dtype: object

In [30]:
nome=trips['name']

a=nome[0]
a='ZTBUS_compressed/'+a+'.csv'
a

'ZTBUS_compressed/B183_2019-04-30_03-18-56_2019-04-30_08-44-20.csv'

In [31]:
tr=pd.read_csv(a, usecols=['itcs_stopName'])

In [32]:
tr

itcs_stopName
0                 -
1                 -
2                 -
3                 -
4                 -
...             ...
19520             -
19521             -
19522             -
19523             -
19524             -

[19525 rows x 1 columns]

In [33]:
%timeit
trips['StopList']=0
nome=trips['name']
for i in range(len(trips)):
    nome[i]='ZTBUS_compressed/'+nome[i]+'.csv'
    a=pd.read_csv(nome[i], usecols=['itcs_stopName'])
    stoplist=a['itcs_stopName'].unique()
    trips['StopList'][i]=stoplist
trips

/var/folders/y0/c69z4ct1695f_69pbyjfvw3h0000gn/T/ipykernel_20791/774854288.py:5: SettingWithCopyWarning: 
A value is trying to be set on a copy of a slice from a DataFrame

See the caveats in the documentation: https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy
  nome[i]='ZTBUS_compressed/'+nome[i]+'.csv'
/var/folders/y0/c69z4ct1695f_69pbyjfvw3h0000gn/T/ipykernel_20791/774854288.py:8: SettingWithCopyWarning: 
A value is trying to be set on a copy of a slice from a DataFrame

See the caveats in the documentation: https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy
  trips['StopList'][i]=stoplist
/var/folders/y0/c69z4ct1695f_69pbyjfvw3h0000gn/T/ipykernel_20791/774854288.py:8: FutureWarning: Setting an item of incompatible dtype is deprecated and will raise in a future error of pandas. Value '['-' 'Zürich, Herdernstrasse' 'Zürich, Hardplatz' 'Zürich, Güterbahnhof'
 'Zürich, Bäckeranlage' 'Zü

/var/folders/y0/c69z4ct1695f_69pbyjfvw3h0000gn/T/ipykernel_20791/774854288.py:5: SettingWithCopyWarning: 
A value is trying to be set on a copy of a slice from a DataFrame

See the caveats in the documentation: https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy
  nome[i]='ZTBUS_compressed/'+nome[i]+'.csv'
/var/folders/y0/c69z4ct1695f_69pbyjfvw3h0000gn/T/ipykernel_20791/774854288.py:5: SettingWithCopyWarning: 
A value is trying to be set on a copy of a slice from a DataFrame

See the caveats in the documentation: https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy
  nome[i]='ZTBUS_compressed/'+nome[i]+'.csv'
/var/folders/y0/c69z4ct1695f_69pbyjfvw3h0000gn/T/ipykernel_20791/774854288.py:5: SettingWithCopyWarning: 
A value is trying to be set on a copy of a slice from a DataFrame

See the caveats in the documentation: https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#retur

/var/folders/y0/c69z4ct1695f_69pbyjfvw3h0000gn/T/ipykernel_20791/774854288.py:5: SettingWithCopyWarning: 
A value is trying to be set on a copy of a slice from a DataFrame

See the caveats in the documentation: https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy
  nome[i]='ZTBUS_compressed/'+nome[i]+'.csv'
/var/folders/y0/c69z4ct1695f_69pbyjfvw3h0000gn/T/ipykernel_20791/774854288.py:5: SettingWithCopyWarning: 
A value is trying to be set on a copy of a slice from a DataFrame

See the caveats in the documentation: https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy
  nome[i]='ZTBUS_compressed/'+nome[i]+'.csv'
/var/folders/y0/c69z4ct1695f_69pbyjfvw3h0000gn/T/ipykernel_20791/774854288.py:5: SettingWithCopyWarning: 
A value is trying to be set on a copy of a slice from a DataFrame

See the caveats in the documentation: https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#retur

/var/folders/y0/c69z4ct1695f_69pbyjfvw3h0000gn/T/ipykernel_20791/774854288.py:5: SettingWithCopyWarning: 
A value is trying to be set on a copy of a slice from a DataFrame

See the caveats in the documentation: https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy
  nome[i]='ZTBUS_compressed/'+nome[i]+'.csv'
/var/folders/y0/c69z4ct1695f_69pbyjfvw3h0000gn/T/ipykernel_20791/774854288.py:5: SettingWithCopyWarning: 
A value is trying to be set on a copy of a slice from a DataFrame

See the caveats in the documentation: https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy
  nome[i]='ZTBUS_compressed/'+nome[i]+'.csv'
/var/folders/y0/c69z4ct1695f_69pbyjfvw3h0000gn/T/ipykernel_20791/774854288.py:5: SettingWithCopyWarning: 
A value is trying to be set on a copy of a slice from a DataFrame

See the caveats in the documentation: https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#retur

/var/folders/y0/c69z4ct1695f_69pbyjfvw3h0000gn/T/ipykernel_20791/774854288.py:5: SettingWithCopyWarning: 
A value is trying to be set on a copy of a slice from a DataFrame

See the caveats in the documentation: https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy
  nome[i]='ZTBUS_compressed/'+nome[i]+'.csv'
/var/folders/y0/c69z4ct1695f_69pbyjfvw3h0000gn/T/ipykernel_20791/774854288.py:5: SettingWithCopyWarning: 
A value is trying to be set on a copy of a slice from a DataFrame

See the caveats in the documentation: https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy
  nome[i]='ZTBUS_compressed/'+nome[i]+'.csv'
/var/folders/y0/c69z4ct1695f_69pbyjfvw3h0000gn/T/ipykernel_20791/774854288.py:5: SettingWithCopyWarning: 
A value is trying to be set on a copy of a slice from a DataFrame

See the caveats in the documentation: https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#retur

/var/folders/y0/c69z4ct1695f_69pbyjfvw3h0000gn/T/ipykernel_20791/774854288.py:5: SettingWithCopyWarning: 
A value is trying to be set on a copy of a slice from a DataFrame

See the caveats in the documentation: https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy
  nome[i]='ZTBUS_compressed/'+nome[i]+'.csv'
/var/folders/y0/c69z4ct1695f_69pbyjfvw3h0000gn/T/ipykernel_20791/774854288.py:5: SettingWithCopyWarning: 
A value is trying to be set on a copy of a slice from a DataFrame

See the caveats in the documentation: https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy
  nome[i]='ZTBUS_compressed/'+nome[i]+'.csv'
/var/folders/y0/c69z4ct1695f_69pbyjfvw3h0000gn/T/ipykernel_20791/774854288.py:5: SettingWithCopyWarning: 
A value is trying to be set on a copy of a slice from a DataFrame

See the caveats in the documentation: https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#retur

/var/folders/y0/c69z4ct1695f_69pbyjfvw3h0000gn/T/ipykernel_20791/774854288.py:5: SettingWithCopyWarning: 
A value is trying to be set on a copy of a slice from a DataFrame

See the caveats in the documentation: https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy
  nome[i]='ZTBUS_compressed/'+nome[i]+'.csv'
/var/folders/y0/c69z4ct1695f_69pbyjfvw3h0000gn/T/ipykernel_20791/774854288.py:5: SettingWithCopyWarning: 
A value is trying to be set on a copy of a slice from a DataFrame

See the caveats in the documentation: https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy
  nome[i]='ZTBUS_compressed/'+nome[i]+'.csv'
/var/folders/y0/c69z4ct1695f_69pbyjfvw3h0000gn/T/ipykernel_20791/774854288.py:5: SettingWithCopyWarning: 
A value is trying to be set on a copy of a slice from a DataFrame

See the caveats in the documentation: https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#retur

/var/folders/y0/c69z4ct1695f_69pbyjfvw3h0000gn/T/ipykernel_20791/774854288.py:5: SettingWithCopyWarning: 
A value is trying to be set on a copy of a slice from a DataFrame

See the caveats in the documentation: https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy
  nome[i]='ZTBUS_compressed/'+nome[i]+'.csv'
/var/folders/y0/c69z4ct1695f_69pbyjfvw3h0000gn/T/ipykernel_20791/774854288.py:5: SettingWithCopyWarning: 
A value is trying to be set on a copy of a slice from a DataFrame

See the caveats in the documentation: https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy
  nome[i]='ZTBUS_compressed/'+nome[i]+'.csv'
/var/folders/y0/c69z4ct1695f_69pbyjfvw3h0000gn/T/ipykernel_20791/774854288.py:5: SettingWithCopyWarning: 
A value is trying to be set on a copy of a slice from a DataFrame

See the caveats in the documentation: https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#retur

/var/folders/y0/c69z4ct1695f_69pbyjfvw3h0000gn/T/ipykernel_20791/774854288.py:5: SettingWithCopyWarning: 
A value is trying to be set on a copy of a slice from a DataFrame

See the caveats in the documentation: https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy
  nome[i]='ZTBUS_compressed/'+nome[i]+'.csv'
/var/folders/y0/c69z4ct1695f_69pbyjfvw3h0000gn/T/ipykernel_20791/774854288.py:5: SettingWithCopyWarning: 
A value is trying to be set on a copy of a slice from a DataFrame

See the caveats in the documentation: https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy
  nome[i]='ZTBUS_compressed/'+nome[i]+'.csv'
/var/folders/y0/c69z4ct1695f_69pbyjfvw3h0000gn/T/ipykernel_20791/774854288.py:5: SettingWithCopyWarning: 
A value is trying to be set on a copy of a slice from a DataFrame

See the caveats in the documentation: https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#retur

/var/folders/y0/c69z4ct1695f_69pbyjfvw3h0000gn/T/ipykernel_20791/774854288.py:5: SettingWithCopyWarning: 
A value is trying to be set on a copy of a slice from a DataFrame

See the caveats in the documentation: https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy
  nome[i]='ZTBUS_compressed/'+nome[i]+'.csv'
/var/folders/y0/c69z4ct1695f_69pbyjfvw3h0000gn/T/ipykernel_20791/774854288.py:5: SettingWithCopyWarning: 
A value is trying to be set on a copy of a slice from a DataFrame

See the caveats in the documentation: https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy
  nome[i]='ZTBUS_compressed/'+nome[i]+'.csv'
/var/folders/y0/c69z4ct1695f_69pbyjfvw3h0000gn/T/ipykernel_20791/774854288.py:5: SettingWithCopyWarning: 
A value is trying to be set on a copy of a slice from a DataFrame

See the caveats in the documentation: https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#retur

/var/folders/y0/c69z4ct1695f_69pbyjfvw3h0000gn/T/ipykernel_20791/774854288.py:5: SettingWithCopyWarning: 
A value is trying to be set on a copy of a slice from a DataFrame

See the caveats in the documentation: https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy
  nome[i]='ZTBUS_compressed/'+nome[i]+'.csv'
/var/folders/y0/c69z4ct1695f_69pbyjfvw3h0000gn/T/ipykernel_20791/774854288.py:5: SettingWithCopyWarning: 
A value is trying to be set on a copy of a slice from a DataFrame

See the caveats in the documentation: https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy
  nome[i]='ZTBUS_compressed/'+nome[i]+'.csv'
/var/folders/y0/c69z4ct1695f_69pbyjfvw3h0000gn/T/ipykernel_20791/774854288.py:5: SettingWithCopyWarning: 
A value is trying to be set on a copy of a slice from a DataFrame

See the caveats in the documentation: https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#retur

/var/folders/y0/c69z4ct1695f_69pbyjfvw3h0000gn/T/ipykernel_20791/774854288.py:5: SettingWithCopyWarning: 
A value is trying to be set on a copy of a slice from a DataFrame

See the caveats in the documentation: https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy
  nome[i]='ZTBUS_compressed/'+nome[i]+'.csv'
/var/folders/y0/c69z4ct1695f_69pbyjfvw3h0000gn/T/ipykernel_20791/774854288.py:5: SettingWithCopyWarning: 
A value is trying to be set on a copy of a slice from a DataFrame

See the caveats in the documentation: https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy
  nome[i]='ZTBUS_compressed/'+nome[i]+'.csv'
/var/folders/y0/c69z4ct1695f_69pbyjfvw3h0000gn/T/ipykernel_20791/774854288.py:5: SettingWithCopyWarning: 
A value is trying to be set on a copy of a slice from a DataFrame

See the caveats in the documentation: https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#retur

/var/folders/y0/c69z4ct1695f_69pbyjfvw3h0000gn/T/ipykernel_20791/774854288.py:5: SettingWithCopyWarning: 
A value is trying to be set on a copy of a slice from a DataFrame

See the caveats in the documentation: https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy
  nome[i]='ZTBUS_compressed/'+nome[i]+'.csv'
/var/folders/y0/c69z4ct1695f_69pbyjfvw3h0000gn/T/ipykernel_20791/774854288.py:5: SettingWithCopyWarning: 
A value is trying to be set on a copy of a slice from a DataFrame

See the caveats in the documentation: https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy
  nome[i]='ZTBUS_compressed/'+nome[i]+'.csv'
/var/folders/y0/c69z4ct1695f_69pbyjfvw3h0000gn/T/ipykernel_20791/774854288.py:5: SettingWithCopyWarning: 
A value is trying to be set on a copy of a slice from a DataFrame

See the caveats in the documentation: https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#retur

/var/folders/y0/c69z4ct1695f_69pbyjfvw3h0000gn/T/ipykernel_20791/774854288.py:5: SettingWithCopyWarning: 
A value is trying to be set on a copy of a slice from a DataFrame

See the caveats in the documentation: https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy
  nome[i]='ZTBUS_compressed/'+nome[i]+'.csv'
/var/folders/y0/c69z4ct1695f_69pbyjfvw3h0000gn/T/ipykernel_20791/774854288.py:5: SettingWithCopyWarning: 
A value is trying to be set on a copy of a slice from a DataFrame

See the caveats in the documentation: https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy
  nome[i]='ZTBUS_compressed/'+nome[i]+'.csv'
/var/folders/y0/c69z4ct1695f_69pbyjfvw3h0000gn/T/ipykernel_20791/774854288.py:5: SettingWithCopyWarning: 
A value is trying to be set on a copy of a slice from a DataFrame

See the caveats in the documentation: https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#retur

/var/folders/y0/c69z4ct1695f_69pbyjfvw3h0000gn/T/ipykernel_20791/774854288.py:5: SettingWithCopyWarning: 
A value is trying to be set on a copy of a slice from a DataFrame

See the caveats in the documentation: https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy
  nome[i]='ZTBUS_compressed/'+nome[i]+'.csv'
/var/folders/y0/c69z4ct1695f_69pbyjfvw3h0000gn/T/ipykernel_20791/774854288.py:5: SettingWithCopyWarning: 
A value is trying to be set on a copy of a slice from a DataFrame

See the caveats in the documentation: https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy
  nome[i]='ZTBUS_compressed/'+nome[i]+'.csv'
/var/folders/y0/c69z4ct1695f_69pbyjfvw3h0000gn/T/ipykernel_20791/774854288.py:5: SettingWithCopyWarning: 
A value is trying to be set on a copy of a slice from a DataFrame

See the caveats in the documentation: https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#retur

/var/folders/y0/c69z4ct1695f_69pbyjfvw3h0000gn/T/ipykernel_20791/774854288.py:5: SettingWithCopyWarning: 
A value is trying to be set on a copy of a slice from a DataFrame

See the caveats in the documentation: https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy
  nome[i]='ZTBUS_compressed/'+nome[i]+'.csv'
/var/folders/y0/c69z4ct1695f_69pbyjfvw3h0000gn/T/ipykernel_20791/774854288.py:5: SettingWithCopyWarning: 
A value is trying to be set on a copy of a slice from a DataFrame

See the caveats in the documentation: https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy
  nome[i]='ZTBUS_compressed/'+nome[i]+'.csv'
/var/folders/y0/c69z4ct1695f_69pbyjfvw3h0000gn/T/ipykernel_20791/774854288.py:5: SettingWithCopyWarning: 
A value is trying to be set on a copy of a slice from a DataFrame

See the caveats in the documentation: https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#retur

/var/folders/y0/c69z4ct1695f_69pbyjfvw3h0000gn/T/ipykernel_20791/774854288.py:5: SettingWithCopyWarning: 
A value is trying to be set on a copy of a slice from a DataFrame

See the caveats in the documentation: https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy
  nome[i]='ZTBUS_compressed/'+nome[i]+'.csv'
/var/folders/y0/c69z4ct1695f_69pbyjfvw3h0000gn/T/ipykernel_20791/774854288.py:5: SettingWithCopyWarning: 
A value is trying to be set on a copy of a slice from a DataFrame

See the caveats in the documentation: https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy
  nome[i]='ZTBUS_compressed/'+nome[i]+'.csv'
/var/folders/y0/c69z4ct1695f_69pbyjfvw3h0000gn/T/ipykernel_20791/774854288.py:5: SettingWithCopyWarning: 
A value is trying to be set on a copy of a slice from a DataFrame

See the caveats in the documentation: https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#retur

/var/folders/y0/c69z4ct1695f_69pbyjfvw3h0000gn/T/ipykernel_20791/774854288.py:5: SettingWithCopyWarning: 
A value is trying to be set on a copy of a slice from a DataFrame

See the caveats in the documentation: https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy
  nome[i]='ZTBUS_compressed/'+nome[i]+'.csv'
/var/folders/y0/c69z4ct1695f_69pbyjfvw3h0000gn/T/ipykernel_20791/774854288.py:5: SettingWithCopyWarning: 
A value is trying to be set on a copy of a slice from a DataFrame

See the caveats in the documentation: https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy
  nome[i]='ZTBUS_compressed/'+nome[i]+'.csv'
/var/folders/y0/c69z4ct1695f_69pbyjfvw3h0000gn/T/ipykernel_20791/774854288.py:5: SettingWithCopyWarning: 
A value is trying to be set on a copy of a slice from a DataFrame

See the caveats in the documentation: https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#retur

/var/folders/y0/c69z4ct1695f_69pbyjfvw3h0000gn/T/ipykernel_20791/774854288.py:5: SettingWithCopyWarning: 
A value is trying to be set on a copy of a slice from a DataFrame

See the caveats in the documentation: https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy
  nome[i]='ZTBUS_compressed/'+nome[i]+'.csv'
/var/folders/y0/c69z4ct1695f_69pbyjfvw3h0000gn/T/ipykernel_20791/774854288.py:5: SettingWithCopyWarning: 
A value is trying to be set on a copy of a slice from a DataFrame

See the caveats in the documentation: https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy
  nome[i]='ZTBUS_compressed/'+nome[i]+'.csv'
/var/folders/y0/c69z4ct1695f_69pbyjfvw3h0000gn/T/ipykernel_20791/774854288.py:5: SettingWithCopyWarning: 
A value is trying to be set on a copy of a slice from a DataFrame

See the caveats in the documentation: https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#retur

/var/folders/y0/c69z4ct1695f_69pbyjfvw3h0000gn/T/ipykernel_20791/774854288.py:5: SettingWithCopyWarning: 
A value is trying to be set on a copy of a slice from a DataFrame

See the caveats in the documentation: https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy
  nome[i]='ZTBUS_compressed/'+nome[i]+'.csv'
/var/folders/y0/c69z4ct1695f_69pbyjfvw3h0000gn/T/ipykernel_20791/774854288.py:5: SettingWithCopyWarning: 
A value is trying to be set on a copy of a slice from a DataFrame

See the caveats in the documentation: https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy
  nome[i]='ZTBUS_compressed/'+nome[i]+'.csv'
/var/folders/y0/c69z4ct1695f_69pbyjfvw3h0000gn/T/ipykernel_20791/774854288.py:5: SettingWithCopyWarning: 
A value is trying to be set on a copy of a slice from a DataFrame

See the caveats in the documentation: https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#retur

/var/folders/y0/c69z4ct1695f_69pbyjfvw3h0000gn/T/ipykernel_20791/774854288.py:5: SettingWithCopyWarning: 
A value is trying to be set on a copy of a slice from a DataFrame

See the caveats in the documentation: https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy
  nome[i]='ZTBUS_compressed/'+nome[i]+'.csv'
/var/folders/y0/c69z4ct1695f_69pbyjfvw3h0000gn/T/ipykernel_20791/774854288.py:5: SettingWithCopyWarning: 
A value is trying to be set on a copy of a slice from a DataFrame

See the caveats in the documentation: https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy
  nome[i]='ZTBUS_compressed/'+nome[i]+'.csv'
/var/folders/y0/c69z4ct1695f_69pbyjfvw3h0000gn/T/ipykernel_20791/774854288.py:5: SettingWithCopyWarning: 
A value is trying to be set on a copy of a slice from a DataFrame

See the caveats in the documentation: https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#retur

/var/folders/y0/c69z4ct1695f_69pbyjfvw3h0000gn/T/ipykernel_20791/774854288.py:5: SettingWithCopyWarning: 
A value is trying to be set on a copy of a slice from a DataFrame

See the caveats in the documentation: https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy
  nome[i]='ZTBUS_compressed/'+nome[i]+'.csv'
/var/folders/y0/c69z4ct1695f_69pbyjfvw3h0000gn/T/ipykernel_20791/774854288.py:5: SettingWithCopyWarning: 
A value is trying to be set on a copy of a slice from a DataFrame

See the caveats in the documentation: https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy
  nome[i]='ZTBUS_compressed/'+nome[i]+'.csv'
/var/folders/y0/c69z4ct1695f_69pbyjfvw3h0000gn/T/ipykernel_20791/774854288.py:5: SettingWithCopyWarning: 
A value is trying to be set on a copy of a slice from a DataFrame

See the caveats in the documentation: https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#retur

/var/folders/y0/c69z4ct1695f_69pbyjfvw3h0000gn/T/ipykernel_20791/774854288.py:5: SettingWithCopyWarning: 
A value is trying to be set on a copy of a slice from a DataFrame

See the caveats in the documentation: https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy
  nome[i]='ZTBUS_compressed/'+nome[i]+'.csv'
/var/folders/y0/c69z4ct1695f_69pbyjfvw3h0000gn/T/ipykernel_20791/774854288.py:5: SettingWithCopyWarning: 
A value is trying to be set on a copy of a slice from a DataFrame

See the caveats in the documentation: https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy
  nome[i]='ZTBUS_compressed/'+nome[i]+'.csv'
/var/folders/y0/c69z4ct1695f_69pbyjfvw3h0000gn/T/ipykernel_20791/774854288.py:5: SettingWithCopyWarning: 
A value is trying to be set on a copy of a slice from a DataFrame

See the caveats in the documentation: https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#retur

/var/folders/y0/c69z4ct1695f_69pbyjfvw3h0000gn/T/ipykernel_20791/774854288.py:5: SettingWithCopyWarning: 
A value is trying to be set on a copy of a slice from a DataFrame

See the caveats in the documentation: https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy
  nome[i]='ZTBUS_compressed/'+nome[i]+'.csv'
/var/folders/y0/c69z4ct1695f_69pbyjfvw3h0000gn/T/ipykernel_20791/774854288.py:5: SettingWithCopyWarning: 
A value is trying to be set on a copy of a slice from a DataFrame

See the caveats in the documentation: https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy
  nome[i]='ZTBUS_compressed/'+nome[i]+'.csv'
/var/folders/y0/c69z4ct1695f_69pbyjfvw3h0000gn/T/ipykernel_20791/774854288.py:5: SettingWithCopyWarning: 
A value is trying to be set on a copy of a slice from a DataFrame

See the caveats in the documentation: https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#retur

/var/folders/y0/c69z4ct1695f_69pbyjfvw3h0000gn/T/ipykernel_20791/774854288.py:5: SettingWithCopyWarning: 
A value is trying to be set on a copy of a slice from a DataFrame

See the caveats in the documentation: https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy
  nome[i]='ZTBUS_compressed/'+nome[i]+'.csv'
/var/folders/y0/c69z4ct1695f_69pbyjfvw3h0000gn/T/ipykernel_20791/774854288.py:5: SettingWithCopyWarning: 
A value is trying to be set on a copy of a slice from a DataFrame

See the caveats in the documentation: https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy
  nome[i]='ZTBUS_compressed/'+nome[i]+'.csv'
/var/folders/y0/c69z4ct1695f_69pbyjfvw3h0000gn/T/ipykernel_20791/774854288.py:5: SettingWithCopyWarning: 
A value is trying to be set on a copy of a slice from a DataFrame

See the caveats in the documentation: https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#retur

/var/folders/y0/c69z4ct1695f_69pbyjfvw3h0000gn/T/ipykernel_20791/774854288.py:5: SettingWithCopyWarning: 
A value is trying to be set on a copy of a slice from a DataFrame

See the caveats in the documentation: https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy
  nome[i]='ZTBUS_compressed/'+nome[i]+'.csv'
/var/folders/y0/c69z4ct1695f_69pbyjfvw3h0000gn/T/ipykernel_20791/774854288.py:5: SettingWithCopyWarning: 
A value is trying to be set on a copy of a slice from a DataFrame

See the caveats in the documentation: https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy
  nome[i]='ZTBUS_compressed/'+nome[i]+'.csv'
/var/folders/y0/c69z4ct1695f_69pbyjfvw3h0000gn/T/ipykernel_20791/774854288.py:5: SettingWithCopyWarning: 
A value is trying to be set on a copy of a slice from a DataFrame

See the caveats in the documentation: https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#retur

/var/folders/y0/c69z4ct1695f_69pbyjfvw3h0000gn/T/ipykernel_20791/774854288.py:5: SettingWithCopyWarning: 
A value is trying to be set on a copy of a slice from a DataFrame

See the caveats in the documentation: https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy
  nome[i]='ZTBUS_compressed/'+nome[i]+'.csv'
/var/folders/y0/c69z4ct1695f_69pbyjfvw3h0000gn/T/ipykernel_20791/774854288.py:5: SettingWithCopyWarning: 
A value is trying to be set on a copy of a slice from a DataFrame

See the caveats in the documentation: https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy
  nome[i]='ZTBUS_compressed/'+nome[i]+'.csv'
/var/folders/y0/c69z4ct1695f_69pbyjfvw3h0000gn/T/ipykernel_20791/774854288.py:5: SettingWithCopyWarning: 
A value is trying to be set on a copy of a slice from a DataFrame

See the caveats in the documentation: https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#retur

/var/folders/y0/c69z4ct1695f_69pbyjfvw3h0000gn/T/ipykernel_20791/774854288.py:5: SettingWithCopyWarning: 
A value is trying to be set on a copy of a slice from a DataFrame

See the caveats in the documentation: https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy
  nome[i]='ZTBUS_compressed/'+nome[i]+'.csv'
/var/folders/y0/c69z4ct1695f_69pbyjfvw3h0000gn/T/ipykernel_20791/774854288.py:5: SettingWithCopyWarning: 
A value is trying to be set on a copy of a slice from a DataFrame

See the caveats in the documentation: https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy
  nome[i]='ZTBUS_compressed/'+nome[i]+'.csv'
/var/folders/y0/c69z4ct1695f_69pbyjfvw3h0000gn/T/ipykernel_20791/774854288.py:5: SettingWithCopyWarning: 
A value is trying to be set on a copy of a slice from a DataFrame

See the caveats in the documentation: https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#retur

/var/folders/y0/c69z4ct1695f_69pbyjfvw3h0000gn/T/ipykernel_20791/774854288.py:5: SettingWithCopyWarning: 
A value is trying to be set on a copy of a slice from a DataFrame

See the caveats in the documentation: https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy
  nome[i]='ZTBUS_compressed/'+nome[i]+'.csv'
/var/folders/y0/c69z4ct1695f_69pbyjfvw3h0000gn/T/ipykernel_20791/774854288.py:5: SettingWithCopyWarning: 
A value is trying to be set on a copy of a slice from a DataFrame

See the caveats in the documentation: https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy
  nome[i]='ZTBUS_compressed/'+nome[i]+'.csv'
/var/folders/y0/c69z4ct1695f_69pbyjfvw3h0000gn/T/ipykernel_20791/774854288.py:5: SettingWithCopyWarning: 
A value is trying to be set on a copy of a slice from a DataFrame

See the caveats in the documentation: https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#retur

/var/folders/y0/c69z4ct1695f_69pbyjfvw3h0000gn/T/ipykernel_20791/774854288.py:5: SettingWithCopyWarning: 
A value is trying to be set on a copy of a slice from a DataFrame

See the caveats in the documentation: https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy
  nome[i]='ZTBUS_compressed/'+nome[i]+'.csv'
/var/folders/y0/c69z4ct1695f_69pbyjfvw3h0000gn/T/ipykernel_20791/774854288.py:5: SettingWithCopyWarning: 
A value is trying to be set on a copy of a slice from a DataFrame

See the caveats in the documentation: https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy
  nome[i]='ZTBUS_compressed/'+nome[i]+'.csv'
/var/folders/y0/c69z4ct1695f_69pbyjfvw3h0000gn/T/ipykernel_20791/774854288.py:5: SettingWithCopyWarning: 
A value is trying to be set on a copy of a slice from a DataFrame

See the caveats in the documentation: https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#retur

/var/folders/y0/c69z4ct1695f_69pbyjfvw3h0000gn/T/ipykernel_20791/774854288.py:5: SettingWithCopyWarning: 
A value is trying to be set on a copy of a slice from a DataFrame

See the caveats in the documentation: https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy
  nome[i]='ZTBUS_compressed/'+nome[i]+'.csv'
/var/folders/y0/c69z4ct1695f_69pbyjfvw3h0000gn/T/ipykernel_20791/774854288.py:5: SettingWithCopyWarning: 
A value is trying to be set on a copy of a slice from a DataFrame

See the caveats in the documentation: https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy
  nome[i]='ZTBUS_compressed/'+nome[i]+'.csv'
/var/folders/y0/c69z4ct1695f_69pbyjfvw3h0000gn/T/ipykernel_20791/774854288.py:5: SettingWithCopyWarning: 
A value is trying to be set on a copy of a slice from a DataFrame

See the caveats in the documentation: https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#retur

/var/folders/y0/c69z4ct1695f_69pbyjfvw3h0000gn/T/ipykernel_20791/774854288.py:5: SettingWithCopyWarning: 
A value is trying to be set on a copy of a slice from a DataFrame

See the caveats in the documentation: https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy
  nome[i]='ZTBUS_compressed/'+nome[i]+'.csv'
/var/folders/y0/c69z4ct1695f_69pbyjfvw3h0000gn/T/ipykernel_20791/774854288.py:5: SettingWithCopyWarning: 
A value is trying to be set on a copy of a slice from a DataFrame

See the caveats in the documentation: https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy
  nome[i]='ZTBUS_compressed/'+nome[i]+'.csv'
/var/folders/y0/c69z4ct1695f_69pbyjfvw3h0000gn/T/ipykernel_20791/774854288.py:5: SettingWithCopyWarning: 
A value is trying to be set on a copy of a slice from a DataFrame

See the caveats in the documentation: https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#retur

/var/folders/y0/c69z4ct1695f_69pbyjfvw3h0000gn/T/ipykernel_20791/774854288.py:5: SettingWithCopyWarning: 
A value is trying to be set on a copy of a slice from a DataFrame

See the caveats in the documentation: https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy
  nome[i]='ZTBUS_compressed/'+nome[i]+'.csv'
/var/folders/y0/c69z4ct1695f_69pbyjfvw3h0000gn/T/ipykernel_20791/774854288.py:5: SettingWithCopyWarning: 
A value is trying to be set on a copy of a slice from a DataFrame

See the caveats in the documentation: https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy
  nome[i]='ZTBUS_compressed/'+nome[i]+'.csv'
/var/folders/y0/c69z4ct1695f_69pbyjfvw3h0000gn/T/ipykernel_20791/774854288.py:5: SettingWithCopyWarning: 
A value is trying to be set on a copy of a slice from a DataFrame

See the caveats in the documentation: https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#retur

/var/folders/y0/c69z4ct1695f_69pbyjfvw3h0000gn/T/ipykernel_20791/774854288.py:5: SettingWithCopyWarning: 
A value is trying to be set on a copy of a slice from a DataFrame

See the caveats in the documentation: https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy
  nome[i]='ZTBUS_compressed/'+nome[i]+'.csv'
/var/folders/y0/c69z4ct1695f_69pbyjfvw3h0000gn/T/ipykernel_20791/774854288.py:5: SettingWithCopyWarning: 
A value is trying to be set on a copy of a slice from a DataFrame

See the caveats in the documentation: https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy
  nome[i]='ZTBUS_compressed/'+nome[i]+'.csv'
/var/folders/y0/c69z4ct1695f_69pbyjfvw3h0000gn/T/ipykernel_20791/774854288.py:5: SettingWithCopyWarning: 
A value is trying to be set on a copy of a slice from a DataFrame

See the caveats in the documentation: https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#retur

/var/folders/y0/c69z4ct1695f_69pbyjfvw3h0000gn/T/ipykernel_20791/774854288.py:5: SettingWithCopyWarning: 
A value is trying to be set on a copy of a slice from a DataFrame

See the caveats in the documentation: https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy
  nome[i]='ZTBUS_compressed/'+nome[i]+'.csv'
/var/folders/y0/c69z4ct1695f_69pbyjfvw3h0000gn/T/ipykernel_20791/774854288.py:5: SettingWithCopyWarning: 
A value is trying to be set on a copy of a slice from a DataFrame

See the caveats in the documentation: https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy
  nome[i]='ZTBUS_compressed/'+nome[i]+'.csv'
/var/folders/y0/c69z4ct1695f_69pbyjfvw3h0000gn/T/ipykernel_20791/774854288.py:5: SettingWithCopyWarning: 
A value is trying to be set on a copy of a slice from a DataFrame

See the caveats in the documentation: https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#retur

/var/folders/y0/c69z4ct1695f_69pbyjfvw3h0000gn/T/ipykernel_20791/774854288.py:5: SettingWithCopyWarning: 
A value is trying to be set on a copy of a slice from a DataFrame

See the caveats in the documentation: https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy
  nome[i]='ZTBUS_compressed/'+nome[i]+'.csv'
/var/folders/y0/c69z4ct1695f_69pbyjfvw3h0000gn/T/ipykernel_20791/774854288.py:5: SettingWithCopyWarning: 
A value is trying to be set on a copy of a slice from a DataFrame

See the caveats in the documentation: https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy
  nome[i]='ZTBUS_compressed/'+nome[i]+'.csv'
/var/folders/y0/c69z4ct1695f_69pbyjfvw3h0000gn/T/ipykernel_20791/774854288.py:5: SettingWithCopyWarning: 
A value is trying to be set on a copy of a slice from a DataFrame

See the caveats in the documentation: https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#retur

/var/folders/y0/c69z4ct1695f_69pbyjfvw3h0000gn/T/ipykernel_20791/774854288.py:5: SettingWithCopyWarning: 
A value is trying to be set on a copy of a slice from a DataFrame

See the caveats in the documentation: https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy
  nome[i]='ZTBUS_compressed/'+nome[i]+'.csv'
/var/folders/y0/c69z4ct1695f_69pbyjfvw3h0000gn/T/ipykernel_20791/774854288.py:5: SettingWithCopyWarning: 
A value is trying to be set on a copy of a slice from a DataFrame

See the caveats in the documentation: https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy
  nome[i]='ZTBUS_compressed/'+nome[i]+'.csv'
/var/folders/y0/c69z4ct1695f_69pbyjfvw3h0000gn/T/ipykernel_20791/774854288.py:5: SettingWithCopyWarning: 
A value is trying to be set on a copy of a slice from a DataFrame

See the caveats in the documentation: https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#retur

/var/folders/y0/c69z4ct1695f_69pbyjfvw3h0000gn/T/ipykernel_20791/774854288.py:5: SettingWithCopyWarning: 
A value is trying to be set on a copy of a slice from a DataFrame

See the caveats in the documentation: https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy
  nome[i]='ZTBUS_compressed/'+nome[i]+'.csv'
/var/folders/y0/c69z4ct1695f_69pbyjfvw3h0000gn/T/ipykernel_20791/774854288.py:5: SettingWithCopyWarning: 
A value is trying to be set on a copy of a slice from a DataFrame

See the caveats in the documentation: https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy
  nome[i]='ZTBUS_compressed/'+nome[i]+'.csv'
/var/folders/y0/c69z4ct1695f_69pbyjfvw3h0000gn/T/ipykernel_20791/774854288.py:5: SettingWithCopyWarning: 
A value is trying to be set on a copy of a slice from a DataFrame

See the caveats in the documentation: https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#retur

/var/folders/y0/c69z4ct1695f_69pbyjfvw3h0000gn/T/ipykernel_20791/774854288.py:5: SettingWithCopyWarning: 
A value is trying to be set on a copy of a slice from a DataFrame

See the caveats in the documentation: https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy
  nome[i]='ZTBUS_compressed/'+nome[i]+'.csv'
/var/folders/y0/c69z4ct1695f_69pbyjfvw3h0000gn/T/ipykernel_20791/774854288.py:5: SettingWithCopyWarning: 
A value is trying to be set on a copy of a slice from a DataFrame

See the caveats in the documentation: https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy
  nome[i]='ZTBUS_compressed/'+nome[i]+'.csv'
/var/folders/y0/c69z4ct1695f_69pbyjfvw3h0000gn/T/ipykernel_20791/774854288.py:5: SettingWithCopyWarning: 
A value is trying to be set on a copy of a slice from a DataFrame

See the caveats in the documentation: https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#retur

/var/folders/y0/c69z4ct1695f_69pbyjfvw3h0000gn/T/ipykernel_20791/774854288.py:5: SettingWithCopyWarning: 
A value is trying to be set on a copy of a slice from a DataFrame

See the caveats in the documentation: https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy
  nome[i]='ZTBUS_compressed/'+nome[i]+'.csv'
/var/folders/y0/c69z4ct1695f_69pbyjfvw3h0000gn/T/ipykernel_20791/774854288.py:5: SettingWithCopyWarning: 
A value is trying to be set on a copy of a slice from a DataFrame

See the caveats in the documentation: https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy
  nome[i]='ZTBUS_compressed/'+nome[i]+'.csv'
/var/folders/y0/c69z4ct1695f_69pbyjfvw3h0000gn/T/ipykernel_20791/774854288.py:5: SettingWithCopyWarning: 
A value is trying to be set on a copy of a slice from a DataFrame

See the caveats in the documentation: https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#retur

/var/folders/y0/c69z4ct1695f_69pbyjfvw3h0000gn/T/ipykernel_20791/774854288.py:5: SettingWithCopyWarning: 
A value is trying to be set on a copy of a slice from a DataFrame

See the caveats in the documentation: https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy
  nome[i]='ZTBUS_compressed/'+nome[i]+'.csv'
/var/folders/y0/c69z4ct1695f_69pbyjfvw3h0000gn/T/ipykernel_20791/774854288.py:5: SettingWithCopyWarning: 
A value is trying to be set on a copy of a slice from a DataFrame

See the caveats in the documentation: https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy
  nome[i]='ZTBUS_compressed/'+nome[i]+'.csv'
/var/folders/y0/c69z4ct1695f_69pbyjfvw3h0000gn/T/ipykernel_20791/774854288.py:5: SettingWithCopyWarning: 
A value is trying to be set on a copy of a slice from a DataFrame

See the caveats in the documentation: https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#retur

/var/folders/y0/c69z4ct1695f_69pbyjfvw3h0000gn/T/ipykernel_20791/774854288.py:5: SettingWithCopyWarning: 
A value is trying to be set on a copy of a slice from a DataFrame

See the caveats in the documentation: https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy
  nome[i]='ZTBUS_compressed/'+nome[i]+'.csv'
/var/folders/y0/c69z4ct1695f_69pbyjfvw3h0000gn/T/ipykernel_20791/774854288.py:5: SettingWithCopyWarning: 
A value is trying to be set on a copy of a slice from a DataFrame

See the caveats in the documentation: https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy
  nome[i]='ZTBUS_compressed/'+nome[i]+'.csv'
/var/folders/y0/c69z4ct1695f_69pbyjfvw3h0000gn/T/ipykernel_20791/774854288.py:5: SettingWithCopyWarning: 
A value is trying to be set on a copy of a slice from a DataFrame

See the caveats in the documentation: https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#retur

/var/folders/y0/c69z4ct1695f_69pbyjfvw3h0000gn/T/ipykernel_20791/774854288.py:5: SettingWithCopyWarning: 
A value is trying to be set on a copy of a slice from a DataFrame

See the caveats in the documentation: https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy
  nome[i]='ZTBUS_compressed/'+nome[i]+'.csv'
/var/folders/y0/c69z4ct1695f_69pbyjfvw3h0000gn/T/ipykernel_20791/774854288.py:5: SettingWithCopyWarning: 
A value is trying to be set on a copy of a slice from a DataFrame

See the caveats in the documentation: https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy
  nome[i]='ZTBUS_compressed/'+nome[i]+'.csv'
/var/folders/y0/c69z4ct1695f_69pbyjfvw3h0000gn/T/ipykernel_20791/774854288.py:5: SettingWithCopyWarning: 
A value is trying to be set on a copy of a slice from a DataFrame

See the caveats in the documentation: https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#retur

/var/folders/y0/c69z4ct1695f_69pbyjfvw3h0000gn/T/ipykernel_20791/774854288.py:5: SettingWithCopyWarning: 
A value is trying to be set on a copy of a slice from a DataFrame

See the caveats in the documentation: https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy
  nome[i]='ZTBUS_compressed/'+nome[i]+'.csv'
/var/folders/y0/c69z4ct1695f_69pbyjfvw3h0000gn/T/ipykernel_20791/774854288.py:5: SettingWithCopyWarning: 
A value is trying to be set on a copy of a slice from a DataFrame

See the caveats in the documentation: https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy
  nome[i]='ZTBUS_compressed/'+nome[i]+'.csv'
/var/folders/y0/c69z4ct1695f_69pbyjfvw3h0000gn/T/ipykernel_20791/774854288.py:5: SettingWithCopyWarning: 
A value is trying to be set on a copy of a slice from a DataFrame

See the caveats in the documentation: https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#retur

/var/folders/y0/c69z4ct1695f_69pbyjfvw3h0000gn/T/ipykernel_20791/774854288.py:5: SettingWithCopyWarning: 
A value is trying to be set on a copy of a slice from a DataFrame

See the caveats in the documentation: https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy
  nome[i]='ZTBUS_compressed/'+nome[i]+'.csv'
/var/folders/y0/c69z4ct1695f_69pbyjfvw3h0000gn/T/ipykernel_20791/774854288.py:5: SettingWithCopyWarning: 
A value is trying to be set on a copy of a slice from a DataFrame

See the caveats in the documentation: https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy
  nome[i]='ZTBUS_compressed/'+nome[i]+'.csv'
/var/folders/y0/c69z4ct1695f_69pbyjfvw3h0000gn/T/ipykernel_20791/774854288.py:5: SettingWithCopyWarning: 
A value is trying to be set on a copy of a slice from a DataFrame

See the caveats in the documentation: https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#retur

/var/folders/y0/c69z4ct1695f_69pbyjfvw3h0000gn/T/ipykernel_20791/774854288.py:5: SettingWithCopyWarning: 
A value is trying to be set on a copy of a slice from a DataFrame

See the caveats in the documentation: https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy
  nome[i]='ZTBUS_compressed/'+nome[i]+'.csv'
/var/folders/y0/c69z4ct1695f_69pbyjfvw3h0000gn/T/ipykernel_20791/774854288.py:5: SettingWithCopyWarning: 
A value is trying to be set on a copy of a slice from a DataFrame

See the caveats in the documentation: https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy
  nome[i]='ZTBUS_compressed/'+nome[i]+'.csv'
/var/folders/y0/c69z4ct1695f_69pbyjfvw3h0000gn/T/ipykernel_20791/774854288.py:5: SettingWithCopyWarning: 
A value is trying to be set on a copy of a slice from a DataFrame

See the caveats in the documentation: https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#retur

/var/folders/y0/c69z4ct1695f_69pbyjfvw3h0000gn/T/ipykernel_20791/774854288.py:5: SettingWithCopyWarning: 
A value is trying to be set on a copy of a slice from a DataFrame

See the caveats in the documentation: https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy
  nome[i]='ZTBUS_compressed/'+nome[i]+'.csv'
/var/folders/y0/c69z4ct1695f_69pbyjfvw3h0000gn/T/ipykernel_20791/774854288.py:5: SettingWithCopyWarning: 
A value is trying to be set on a copy of a slice from a DataFrame

See the caveats in the documentation: https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy
  nome[i]='ZTBUS_compressed/'+nome[i]+'.csv'
/var/folders/y0/c69z4ct1695f_69pbyjfvw3h0000gn/T/ipykernel_20791/774854288.py:5: SettingWithCopyWarning: 
A value is trying to be set on a copy of a slice from a DataFrame

See the caveats in the documentation: https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#retur

/var/folders/y0/c69z4ct1695f_69pbyjfvw3h0000gn/T/ipykernel_20791/774854288.py:5: SettingWithCopyWarning: 
A value is trying to be set on a copy of a slice from a DataFrame

See the caveats in the documentation: https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy
  nome[i]='ZTBUS_compressed/'+nome[i]+'.csv'
/var/folders/y0/c69z4ct1695f_69pbyjfvw3h0000gn/T/ipykernel_20791/774854288.py:5: SettingWithCopyWarning: 
A value is trying to be set on a copy of a slice from a DataFrame

See the caveats in the documentation: https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy
  nome[i]='ZTBUS_compressed/'+nome[i]+'.csv'
/var/folders/y0/c69z4ct1695f_69pbyjfvw3h0000gn/T/ipykernel_20791/774854288.py:5: SettingWithCopyWarning: 
A value is trying to be set on a copy of a slice from a DataFrame

See the caveats in the documentation: https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#retur

/var/folders/y0/c69z4ct1695f_69pbyjfvw3h0000gn/T/ipykernel_20791/774854288.py:5: SettingWithCopyWarning: 
A value is trying to be set on a copy of a slice from a DataFrame

See the caveats in the documentation: https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy
  nome[i]='ZTBUS_compressed/'+nome[i]+'.csv'
/var/folders/y0/c69z4ct1695f_69pbyjfvw3h0000gn/T/ipykernel_20791/774854288.py:5: SettingWithCopyWarning: 
A value is trying to be set on a copy of a slice from a DataFrame

See the caveats in the documentation: https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy
  nome[i]='ZTBUS_compressed/'+nome[i]+'.csv'
/var/folders/y0/c69z4ct1695f_69pbyjfvw3h0000gn/T/ipykernel_20791/774854288.py:5: SettingWithCopyWarning: 
A value is trying to be set on a copy of a slice from a DataFrame

See the caveats in the documentation: https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#retur

/var/folders/y0/c69z4ct1695f_69pbyjfvw3h0000gn/T/ipykernel_20791/774854288.py:5: SettingWithCopyWarning: 
A value is trying to be set on a copy of a slice from a DataFrame

See the caveats in the documentation: https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy
  nome[i]='ZTBUS_compressed/'+nome[i]+'.csv'
/var/folders/y0/c69z4ct1695f_69pbyjfvw3h0000gn/T/ipykernel_20791/774854288.py:5: SettingWithCopyWarning: 
A value is trying to be set on a copy of a slice from a DataFrame

See the caveats in the documentation: https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy
  nome[i]='ZTBUS_compressed/'+nome[i]+'.csv'
/var/folders/y0/c69z4ct1695f_69pbyjfvw3h0000gn/T/ipykernel_20791/774854288.py:5: SettingWithCopyWarning: 
A value is trying to be set on a copy of a slice from a DataFrame

See the caveats in the documentation: https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#retur

/var/folders/y0/c69z4ct1695f_69pbyjfvw3h0000gn/T/ipykernel_20791/774854288.py:5: SettingWithCopyWarning: 
A value is trying to be set on a copy of a slice from a DataFrame

See the caveats in the documentation: https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy
  nome[i]='ZTBUS_compressed/'+nome[i]+'.csv'
/var/folders/y0/c69z4ct1695f_69pbyjfvw3h0000gn/T/ipykernel_20791/774854288.py:5: SettingWithCopyWarning: 
A value is trying to be set on a copy of a slice from a DataFrame

See the caveats in the documentation: https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy
  nome[i]='ZTBUS_compressed/'+nome[i]+'.csv'
/var/folders/y0/c69z4ct1695f_69pbyjfvw3h0000gn/T/ipykernel_20791/774854288.py:5: SettingWithCopyWarning: 
A value is trying to be set on a copy of a slice from a DataFrame

See the caveats in the documentation: https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#retur

/var/folders/y0/c69z4ct1695f_69pbyjfvw3h0000gn/T/ipykernel_20791/774854288.py:5: SettingWithCopyWarning: 
A value is trying to be set on a copy of a slice from a DataFrame

See the caveats in the documentation: https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy
  nome[i]='ZTBUS_compressed/'+nome[i]+'.csv'
/var/folders/y0/c69z4ct1695f_69pbyjfvw3h0000gn/T/ipykernel_20791/774854288.py:5: SettingWithCopyWarning: 
A value is trying to be set on a copy of a slice from a DataFrame

See the caveats in the documentation: https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy
  nome[i]='ZTBUS_compressed/'+nome[i]+'.csv'
/var/folders/y0/c69z4ct1695f_69pbyjfvw3h0000gn/T/ipykernel_20791/774854288.py:5: SettingWithCopyWarning: 
A value is trying to be set on a copy of a slice from a DataFrame

See the caveats in the documentation: https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#retur

/var/folders/y0/c69z4ct1695f_69pbyjfvw3h0000gn/T/ipykernel_20791/774854288.py:5: SettingWithCopyWarning: 
A value is trying to be set on a copy of a slice from a DataFrame

See the caveats in the documentation: https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy
  nome[i]='ZTBUS_compressed/'+nome[i]+'.csv'
/var/folders/y0/c69z4ct1695f_69pbyjfvw3h0000gn/T/ipykernel_20791/774854288.py:5: SettingWithCopyWarning: 
A value is trying to be set on a copy of a slice from a DataFrame

See the caveats in the documentation: https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy
  nome[i]='ZTBUS_compressed/'+nome[i]+'.csv'
/var/folders/y0/c69z4ct1695f_69pbyjfvw3h0000gn/T/ipykernel_20791/774854288.py:5: SettingWithCopyWarning: 
A value is trying to be set on a copy of a slice from a DataFrame

See the caveats in the documentation: https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#retur

/var/folders/y0/c69z4ct1695f_69pbyjfvw3h0000gn/T/ipykernel_20791/774854288.py:5: SettingWithCopyWarning: 
A value is trying to be set on a copy of a slice from a DataFrame

See the caveats in the documentation: https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy
  nome[i]='ZTBUS_compressed/'+nome[i]+'.csv'
/var/folders/y0/c69z4ct1695f_69pbyjfvw3h0000gn/T/ipykernel_20791/774854288.py:5: SettingWithCopyWarning: 
A value is trying to be set on a copy of a slice from a DataFrame

See the caveats in the documentation: https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy
  nome[i]='ZTBUS_compressed/'+nome[i]+'.csv'
/var/folders/y0/c69z4ct1695f_69pbyjfvw3h0000gn/T/ipykernel_20791/774854288.py:5: SettingWithCopyWarning: 
A value is trying to be set on a copy of a slice from a DataFrame

See the caveats in the documentation: https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#retur

/var/folders/y0/c69z4ct1695f_69pbyjfvw3h0000gn/T/ipykernel_20791/774854288.py:5: SettingWithCopyWarning: 
A value is trying to be set on a copy of a slice from a DataFrame

See the caveats in the documentation: https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy
  nome[i]='ZTBUS_compressed/'+nome[i]+'.csv'
/var/folders/y0/c69z4ct1695f_69pbyjfvw3h0000gn/T/ipykernel_20791/774854288.py:5: SettingWithCopyWarning: 
A value is trying to be set on a copy of a slice from a DataFrame

See the caveats in the documentation: https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy
  nome[i]='ZTBUS_compressed/'+nome[i]+'.csv'
/var/folders/y0/c69z4ct1695f_69pbyjfvw3h0000gn/T/ipykernel_20791/774854288.py:5: SettingWithCopyWarning: 
A value is trying to be set on a copy of a slice from a DataFrame

See the caveats in the documentation: https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#retur

name  busNumber  \
0     ZTBUS_compressed/B183_2019-04-30_03-18-56_2019...        183   
1     ZTBUS_compressed/B183_2019-04-30_13-22-07_2019...        183   
2     ZTBUS_compressed/B183_2019-05-01_05-58-51_2019...        183   
3     ZTBUS_compressed/B183_2019-05-03_02-50-21_2019...        183   
4     ZTBUS_compressed/B183_2019-05-03_15-41-57_2019...        183   
...                                                 ...        ...   
1404  ZTBUS_compressed/B208_2022-12-06_14-43-49_2022...        208   
1405  ZTBUS_compressed/B208_2022-12-07_05-13-02_2022...        208   
1406  ZTBUS_compressed/B208_2022-12-08_05-22-20_2022...        208   
1407  ZTBUS_compressed/B208_2022-12-09_23-55-12_2022...        208   
1408  ZTBUS_compressed/B208_2022-12-10_23-34-46_2022...        208   

             startTime_iso  startTime_unix           endTime_iso  \
0     2019-04-30T03:18:56Z      1556594336  2019-04-30T08:44:20Z   
1     2019-04-30T13:22:07Z      1556630527  2019-04-30T17:54:02Z   
2     2019-05-01T05:58:51Z      1556690331  2019-05-01T22:32:30Z   
3     2019-05-03T02:50:21Z      1556851821  2019-05-03T05:53:20Z   
4     2019-05-03T15:41:57Z      1556898117  2019-05-03T23:06:24Z   
...                    ...             ...                   ...   
1404  2022-12-06T14:43:49Z      1670337829  2022-12-06T18:22:52Z   
1405  2022-12-07T05:13:02Z      1670389982  2022-12-07T19:19:53Z   
1406  2022-12-08T05:22:20Z      1670476940  2022-12-08T18:39:15Z   
1407  2022-12-09T23:55:12Z      1670630112  2022-12-10T03:24:28Z   
1408  2022-12-10T23:34:46Z      1670715286  2022-12-11T03:29:05Z   

      endTime_unix  drivenDistance busRoute  energyConsumption  \
0       1556613860        77213.87        -       4.785852e+08   
1       1556646842        59029.60       31       4.022585e+08   
2       1556749950       240900.40       33       1.445733e+09   
3       1556862800        42565.48        -       2.819867e+08   
4       1556924784       125277.20       72       6.207258e+08   
...            ...             ...      ...                ...   
1404    1670350972        51798.78       32       4.260419e+08   
1405    1670440793       210041.60       83       1.536697e+09   
1406    1670524755       190372.70       83       1.415700e+09   
1407    1670642668        59548.57       N1       4.519165e+08   
1408    1670729345        72293.83       N2       5.350956e+08   

      itcs_numberOfPassengers_mean  itcs_numberOfPassengers_min  \
0                         5.538860                            0   
1                        33.114580                            4   
2                        19.689140                            0   
3                         1.685185                            0   
4                        23.753570                            1   
...                            ...                          ...   
1404                     39.808990                            0   
1405                     28.785390                            0   
1406                     29.879400                            0   
1407                     20.105260                            0   
1408                     15.993200                            0   

      itcs_numberOfPassengers_max  status_gridIsAvailable_mean  \
0                              20                     0.740640   
1                              74                     0.855234   
2                              55                     0.777860   
3                               8                     0.767122   
4                              67                     0.907342   
...                           ...                          ...   
1404                           83                     0.739349   
1405                          115                     0.434858   
1406                          102                     0.439916   
1407                           74                     0.495739   
1408                           52                     0.459492   

  

In [34]:
trips['StopList'][0]==test['itcs_stopName'].unique()

array([ True,  True,  True,  True,  True,  True,  True,  True,  True,
        True,  True,  True,  True,  True,  True,  True,  True,  True,
        True,  True,  True,  True,  True,  True,  True,  True,  True,
        True,  True,  True,  True,  True])